conf.js

```diff
@@ -0,0 +1,9 @@
+const  mysql = require('mysql');
+const  connection = mysql.createConnection({
+  host :  'localhost',
+  user :  'angelique',
+  password :  'Bija2413!',
```

Comment:
thx for your password :)

this file should not be pushed... and should be part of a react project